test/scrabble_test.rb

```diff
@@ -6,8 +6,21 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_creates_an_array
+    newword = Scrabble.new("hello")
+    expected = ["h", "e", "l", "l", "o"]
+    actual = newword.break_word_into_an_array("hello")
+    assert_equal expected, actual
+  end
+
+  def test_it_can_score_a_word
+    assert_equal [1,2,1,1,1], Scrabble.new.get_individaul_letters(["h","e","l","l","o"])
```

Comment:
Not a bad test. I would probably call the method `score_individual_letters`, but this feels like the right idea.